### ISSUE TITLE
"Inspecting the Ruins" quest related to Shady Rest Inn in Dustwallow Marsh

### ISSUE THREAD

**Issue by Sidefix** [TrinityCore/TrinityCore#25847]
**Description:**

The related item quests from the scorched inn 'Shady Rest Inn' from the entry to Dustwallow Marsh from the Barrens allow you to pick up the quests related to them (Mysterious Hoofprints, Scorched Shield, Stray Badges) without having the "Reflective Dust" buff from Inspector Tarem.

**Expected behaviour:**

The items should reveal and allow an individual to pick up their respective quests only with the 'Reflective Dust' buff that is given upon completion of the "Inspecting the Ruins" quest. This should then allow an individual to pick up the related item quests to further the investigation.

Reference for expected behavior (see Notes):
https://wow.gamepedia.com/Inspecting_the_Ruins
https://youtu.be/EI3E4pQZ130?t=306

**Steps to reproduce the problem:**

1. Enter Dustwallow Marsh at the appropriate level (approx 40)
2. See the Shady Rest Inn at the entrance to the area
3. Observe that the items from within the burned Inn have immediately available quests; these should only become available as part of a follow-up

**Branch(es):**

3.3.5

**TC rev. hash/commit:** 

03c1fd556de5 2021-01-08 03:31:48

**Operating system:** Ubuntu 20.20 x64

**Comment by offl** [comment]
> to pick up their respective quests only with the 'Reflective Dust' buff that is given upon completion

No, that aura and that NPC with 'Inspecting the Ruins' were added only in 2.3.0 just to point players to objects-questgivers. The aura gives invisibility detection to allow to see invisible evidence markers.

I don't really understand why I can pick quests from objects without pre-quests but so far can't find how it's supposed to work.

**Comment by Alastro-Kraken** [comment]
> No, that aura and that NPC with 'Inspecting the Ruins' were added only in 2.3.0 just to point players to objects-questgivers. The aura gives invisibility detection to allow to see invisible evidence markers.

This is correct, the aura is completely optional, even though it is automatically cast when you turn in the quest Inspecting the Ruins. It is also limited to the Shady Rest Inn area - one step outside that area and you lose it (but you can ask the NPC to recast it on you).

The three quests have no requirement, you can grab a character that never did a single quest in Dustwallow, head to the Inn and grab the three quests from the gameobjects. You won't get the aura to point to the objects, but as I said that one is optional.

Moreover, for the "things that don't make sense!"™ series, you can head to the Inn, grab the three quests from the gameobjects, return to the quest hub, turn them in, THEN take Inspecting the Ruins, return to the Shady Inn, turn it in and get the powder cast on you anytime you want.

All of this tested on TBC's PTR.

Long story short, this issue is not actually a bug, but a feature™.

**Comment by offl** [comment]
Thanks for testing :)
